apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.Tests/EnumMapTest.cs

```diff
@@ -36,13 +37,35 @@ public void InvalidConversion()
             Assert.Throws<ArgumentException>(() => EnumMap<WriteDisposition>.ToValue("WRITEIFEMPTY"));
             Assert.Throws<ArgumentException>(() => EnumMap<WriteDisposition>.ToValue("Bogus"));
             Assert.Throws<ArgumentException>(() => EnumMap<WriteDisposition>.ToValue("Write_Empty"));
-            Assert.Throws<ArgumentException>(() => EnumMap<WriteDisposition>.ToApiValue((WriteDisposition) 100));
+            Assert.Throws<ArgumentException>(() => EnumMap<WriteDisposition>.ToApiValue((WriteDisposition)100));
+            Assert.Throws<ArgumentException>(() => EnumMap<SchemaUpdateOption>.ToApiValues((SchemaUpdateOption)100));
+        }
+
+        [Theory]
```

Comment:
InvalidFlagsConversion test?

Reply:
I added it on the method above InvalidConversion since that was dealing with the two types of coversions  that existed till now, so added one Assert for the ToApiValues conversion.